One-to-One-Video/Agora-Web-Tutorial-1to1-React/src/hooks/useMediaStream.ts

```diff
@@ -12,10 +12,7 @@ const useMediaStream = (client: any, filter?: (streamId: number) => boolean): an
         return;
       }
       const { stream } = evt;
-      setRemoteStreamList(streamList => {
-        streamList.push(stream);
```

Comment:
We need to return the new object otherwise react won't update the state.
<img width="1632" alt="React App 🔊 2019-11-18 10 AM-34-06 png 2019-11-18 10 AM-51-39" src="https://user-images.githubusercontent.com/26645888/69037950-8452ab00-09f1-11ea-8681-d01de8382c63.png">